toolversions.sh

```diff
@@ -42,6 +42,7 @@ install_docfx() {
   if [[ ! -f $DOCFX ]]
   then
     (echo "Fetching docfx v${DOCFX_VERSION}";
+     mkdir -p $REPO_ROOT/packages;
```

Comment:
Have just tried to use nuget install with the package, and it fails to install due to invalid dependencies. Let's stick with the zip file :)